javascript-esm/.nvmrc

```diff
@@ -0,0 +1 @@
+v14.16.1
```

Comment:
[Gallium is now in active LTS](https://nodejs.org/en/about/releases/)

```suggestion
v16.13.0
```

Reply:
I've done this across all three JS/TS projects in https://github.com/guardian/coding-exercise-project/pull/73